tripal/tripal
# Catch adding Chado field to non-Chado content type

# Bug Fix

### Closes #1837

### Tripal Version: 4.x

## Description
This adds a simple check to catch when attempting to add a Chado field to a non-Chado content type. Instead of a WSOD as described in issue #1837, we will now display a simple error message.

## Testing?
  1. Create a new Drupal content type. Go to `/admin/structure/types/add`
  2. Enter a "Name" and then click "Save and manage fields"
  3. Click "+ Add a new field"
  4. Enter a "Label"
  5. Select Chado fields
  6. Pick any Chado field, e.g. analysis
  7. Click "Continue"
 A friendly error message should replace the former WSOD
![2024-04-11_invalid_field_appearance](https://github.com/tripal/tripal/assets/8419404/f4997bb3-0b64-48a1-9040-6dc33e3dbf2c)
